File: scala-gatling-client/src/gatling/scala/io/swagger/client/model/PushDataObject.scala

package io.swagger.client.model


case class PushDataObject (
    _gcm: Option[PushDataObject_gcm],
    _aps: Option[PushDataObject_aps],
    _extras: Option[PushDataObject_extras]
)
object PushDataObject {
    def toStringBody(var_gcm: Object, var_aps: Object, var_extras: Object) =
        s"""
        | {
        | "gcm":$var_gcm,"aps":$var_aps,"extras":$var_extras
        | }
        """.stripMargin
}

File: dart-client/lib/model/push_data_object.dart
part of swagger.api;

class PushDataObject {
  
  PushDataObjectGcm gcm = null;
  

  PushDataObjectAps aps = null;
  

  PushDataObjectExtras extras = null;
  
  PushDataObject();

  @override
  String toString() {
    return 'PushDataObject[gcm=$gcm, aps=$aps, extras=$extras, ]';
  }

  PushDataObject.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    gcm =
      
      
      new PushDataObjectGcm.fromJson(json['gcm'])
;
    aps =
      
      
      new PushDataObjectAps.fromJson(json['aps'])
;
    extras =
      
      
      new PushDataObjectExtras.fromJson(json['extras'])
;
  }

  Map<String, dynamic> toJson() {
    return {
      'gcm': gcm,
      'aps': aps,
      'extras': extras
     };
  }

  static List<PushDataObject> listFromJson(List<dynamic> json) {
    return json == null ? new List<PushDataObject>() : json.map((value) => new PushDataObject.fromJson(value)).toList();
  }

  static Map<String, PushDataObject> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, PushDataObject>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new PushDataObject.fromJson(value));
    }
    return map;
  }
}

